src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/JavaScriptEncoder.cs

```diff
@@ -73,6 +73,9 @@ public DefaultJavaScriptEncoder(TextEncoderSettings filter)
 
             _allowedCharacters.ForbidCharacter('\\');
             _allowedCharacters.ForbidCharacter('/');
+            
+            // Disallow GRAVE ACCEN \u0060 character as it is supported in ECMAScript v6
```

Comment:
nits: `ACCENT` and does "supported" (in generally) really matter? More important that back-tick is a string-quoting character.